samples/kubeflow/install/kubeflow/README.md

```diff
@@ -0,0 +1,34 @@
+## Kubeflow on OpenShift 4+ with Istio 1.7 Installation 
+
+Follow [Kubeflow Documentation](https://www.kubeflow.org/docs/openshift/install-kubeflow/) to install Kubeflow with a few changes detailed below:
+```bash
+git clone https://github.com/opendatahub-io/manifests.git
+```
+
+Changes to `kfctl_openshift.yaml`:
+* Since we already have Istio 1.7 installed, comment out `istio-crds` and `istio-install`
+* Comment out `cert-manager-crds` and `cert-manager` 
+* Comment out `seldon-core-operator` 
+
+A modified `kfctl_openshift.yaml` is provided in this repo.
```

Comment:
I suggest to move this line before the list of changes. 
Something along the lines of "A modified `kfctl_openshift.yaml` is provided in this repo with the following changes..."